api/controllers/site.js

```diff
@@ -27,41 +29,45 @@ const stripCredentials = ({ username, password }) => {
 
 module.exports = wrapHandlers({
   async findAllForUser(req, res) {
-    const sites = await Site.forUser(req.user).findAll();
+    const { user } = req;
+
+    const sites = await Site.forUser(user).findAll();
 
     if (!sites) {
       return res.notFound();
     }
 
-    const siteJSON = await siteSerializer.serialize(sites);
+    const siteJSON = siteSerializer.serializeMany(sites);
```

Comment:
We can update `serializeMany` to use `serializeObject` instead of `serializeNew` because `serializeObject` runs `serializeNew` and then cleans up the included `User` and `Organization` related keys.

Reply:
Probably should change the name to something like `serializeIncluded`. I'll work on it and get you alls take.